### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by CIPop**
Initial cleanup for NameResolution.

### CODE REVIEW

**Comment by CIPop** [comment]
@pgavlin PTAL
This doesn't include manual comment scrubbing. I'm doing that within the NameResolutionPR branch (based on master).

File: src/Common/src/Interop/OSX/libc/Interop.sockaddr.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public struct sockaddr_in
             public sa_family_t sin_family; // Address family: AF_INET
             public in_port_t sin_port;     // Port in network byte order
             public in_addr sin_addr;       // Internet address
-            private ulong padding;         // 8 bytes of padding
+            private ulong _padding;        // 8 bytes of padding
```
Comment (pgavlin) [inline src/Common/src/Interop/OSX/libc/Interop.sockaddr.cs]: This should not be changed: as per the interop guidelines, field names for native structs should be identical to the names in the native headers.

File: src/Common/src/Interop/Unix/libc/Interop.sockaddr.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ public struct sockaddr_in
             public sa_family_t sin_family; // Address family: AF_INET
             public in_port_t sin_port;     // Port in network byte order
             public in_addr sin_addr;       // Internet address
-            private ulong padding;         // 8 bytes of padding
+            private ulong _padding;         // 8 bytes of padding
```
Comment (pgavlin) [inline src/Common/src/Interop/Unix/libc/Interop.sockaddr.cs]: Same as above.

File: src/Common/src/Interop/Windows/Winsock/ProtocolFamily.cs [hunk 3]
```diff
@@ -9,7 +9,7 @@ namespace System.Net.Sockets
     ///       class can use.
     ///    </para>
     /// </devdoc>
-    public enum ProtocolFamily
+    internal enum ProtocolFamily
```
Comment (pgavlin) [inline src/Common/src/Interop/Windows/Winsock/ProtocolFamily.cs:12]: Is this visibility change intentional?

Comment (CIPop) [inline src/Common/src/Interop/Windows/Winsock/ProtocolFamily.cs:12]: Yes, for now, as the type isn't exposed by any contract. There are teams that may require using this type internally - we can switch back to public.
Doing this per our decision to be careful about public types in Common.

File: src/Common/src/Interop/OSX/libc/Interop.sockaddr.cs (hunk 1 above)
Comment (CIPop) [inline src/Common/src/Interop/OSX/libc/Interop.sockaddr.cs]: Will revert here and in the NameResolutionPR.

File: src/Common/src/Interop/Unix/libc/Interop.sockaddr.cs (hunk 2 above)
Comment (CIPop) [inline src/Common/src/Interop/Unix/libc/Interop.sockaddr.cs]: Will revert here and in the NameResolutionPR.

**Pull request merged**